build.gradle.kts

```diff
@@ -1,6 +1,3 @@
-// TODO: change the hardcoded 1.6.10 to kotlinVeresion that lives in project props
-// We could use a buildSrc script for this.
-// See: https://github.com/gradle/kotlin-dsl-samples/issues/802
 plugins {
     kotlin("jvm") version "1.8.10"
```

Comment:
We are still hardcoding 1.8.10 here? Any way we can use the version from the platform project to avoid having this duplicate?

Reply:
Good point. It's not explicit in my initial revision but if you try to use another version that's different to what's constrained in the platform project, it'll complain stating the the lib is not found. I've refactored the plugins with their versions into a centralised place to live in `settings.gradle.kts` with a comment to make it obvious on how it all works. This will make it easier to see what versions we use.